src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/PackageLevelTypeToGroupNameConverter.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows.Data;
+using NuGet.VisualStudio;
+
+namespace NuGet.PackageManagement.UI
+{
+    internal class PackageLevelTypeToGroupNameConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is PackageLevelType pkgLevelType)
+            {
+                if (pkgLevelType == PackageLevelType.TopLevel)
+                {
+                    return Resources.PackageLevelType_TopLevelPackageHeaderText;
+                }
+                else if (pkgLevelType == PackageLevelType.Transitive)
+                {
+                    return Resources.PackageLevelType_TransitivePackageHeaderText;
+                }
+            }
+
+            return null;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is string pkgLevelTypeString)
```

Comment:
How is `ConvertBack` being used? What happens if you don't implement it?

Reply:
@donnie-msft it's not being used in this PR. It was implemented for completeness and backed by tests. It may be used in a subsequent PR, depending on how sorting is handled but it may not.

Reply:
Ok, I suggest removing it for now. It doesn't look like it would be difficult to add back if it's needed in the future.